spec/cli.spec.js

```diff
@@ -55,76 +55,76 @@ describe("cordova cli", function () {
     });
 
     describe("options", function () {
-        describe("version", function () {
-            var version = require("../package").version;
+      describe("version", function () {
```

Comment:
nit: could you please use 4 spaces for indentation to be consistent with the rest of the tests/code?